### PR TITLE
Refactoring of the codebase #116

### PR DESCRIPTION
**Pull request by pointerxr**
related to #116

### CODE REVIEW

**Comment by PeteGoo** [comment]
Looking good. Label with `Ready` when you're happy.

I might be a bit slow today, much working, little sleep

**Comment by pointerxr** [comment]
Ok, I'm heading to sleep now as well.

It's not really done tho.

**Comment by pointerxr** [comment]
Well I think the best would be to do it again from scratch with the code as hint. It's all mixed up, everything is just a big piece of s*** code.

**Comment by PeteGoo** [comment]
Cool. I'm happy to commit to that and I'm happy to build on this if you are. Possibly a meeting of minds conversation needs to happen first though?

**Comment by pointerxr** [comment]
Yep that would be indeed a good idea.

**Comment by pointerxr** [comment]
I'm continuing this by the end of the week since I need to sort some things out in real life

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ addons:
   artifacts:
     # ...
     paths:
-    - $(ls tcslackbuildnotifier-web-ui/target/*.zip | tr "\n" ":")
+    - $(ls target/*.zip | tr "\n" ":")
```
Comment (PeteGoo) [inline .travis.yml]: :heart:

**Pull request merged**